boards/arm/sam4s_xplained/ext_ram.c

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
SMC driver should better be located somewhere in drivers/ folder as it is SoC specific, not board specific.